include/swift/Basic/LangOptions.h

```diff
@@ -332,8 +333,7 @@ namespace swift {
     }
 
   private:
-    llvm::SmallVector<std::pair<PlatformConditionKind, std::string>,
-                      NumPlatformConditionKind>
+    llvm::SmallVector<std::pair<PlatformConditionKind, std::string>, 4>
```

Comment:
It's a bit unfortunate to lose the named constant here...

Reply:
Maybe, but I'm not really sure what it buys us.